doc/configcommands.dsv

```diff
@@ -125,7 +125,7 @@ ttrss-password||<password>||""||Configures the password for use with Tiny Tiny R
 ttrss-passwordfile||<path>||""||A more secure alternative to the above, by storing your password elsewhere in your system.||ttrss-passwordfile "~/.newsboat/ttrss-pw.txt"
 ttrss-passwordeval||<command>||""||Another secure alternative, is providing your password from an external command that is evaluated during login. This can be used to read your password from a gpg encrypted file or your system keyring.||ttrss-passwordeval "command some-parameter"
 ttrss-url||<url>||""||Configures the URL where the Tiny Tiny RSS installation you want to use resides.||ttrss-url "http://example.com/ttrss/"
-unbind-key||<key> [<dialog>]||n/a||Unbind key <key>. This means that no operation is called when <key> is pressed. Optionally, you can specify a dialog (for a list of available dialogs, see `bind-key` above). If you specify one, the key binding will only be unbound for the specified dialog.||unbind-key R
+unbind-key||<key> [<dialog>]||n/a||Unbind key <key>. This means that no operation is called when <key> is pressed. If you provide "-a" as <key>, all currently bound keys will be unbinded. Optionally, you can specify a dialog (for a list of available dialogs, see `bind-key` above). If you specify one, the key binding will only be unbound for the specified dialog.||unbind-key R
```

Comment:
Shouldn't it be `unbound` instead of `unbinded`?

Reply:
Hmm, I think you're right.

Reply:
How about `will become unbound`?

Reply:
I like that even better.